controllers/userController.js

```diff
@@ -19,23 +25,35 @@ export default class UserController {
   }
 
   // Create New User.
-  createUser(req) {
+  async createUser(req) {
     // Get POST params
     const {
       email, phoneNo, fullName, password,
     } = req.body;
 
+    const hashedPassword = bcrypt.hashSync(password, 10);
+
     const newUser = new User(email, fullName,
-      phoneNo, password);
+      phoneNo, hashedPassword, new Date(), roleId);
 
-    const userFound = this.mapUserList.get(email);
-    const status = (userFound === undefined) ? 201 : 409;
+    const userFound = await db.query(userquery.queryUser(email));
+    const status = (userFound.rowCount === 1) ? 409 : 201;
 
     // Populate List in Memory if object is not empty
     if (!(validate(newUser)) && status === 201) {
-      this.mapUserList.set(newUser.email, newUser);
-      this.response = new Response('ok', status, 'New user Created', newUser);
+      const success = await db.query(
+        userquery.createUser(
+          newUser.email, newUser.fullName,
+          newUser.phoneNo, newUser.password,
+          newUser.dateCreated, newUser.roleId,
+        ),
+      );
+      if (success.rowCount === 1) {
+        newUser.password = '';
+        this.response = new Response('ok', status, 'New user Created', newUser);
+      }
     } else {
+      newUser.password = '';
       this.response = new Response('ok', status, 'User Already Exists, Consider Logging In', newUser);
     }
     // res.status(status).send(response).end();
```

Comment:
Why is this commented out?

Reply:
its commented out because a standard format for response is used. The response contains status, code , message and result. The router method uses the object instead of the controller.